spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/sending-messages.adoc

```diff
@@ -650,6 +650,12 @@ public Message<?> messageReturn(String in) {
 }
 ----
 
+=== Original Record Key in Reply
```

Comment:
Why do you still think that this short paragraph deserves its own title?

Reply:
I don't see a problem with a separate section for this, but I am neutral either way.